heapster-saw/examples/Makefile

```diff
@@ -19,19 +19,19 @@ endif
 ifeq ($(CI),)
```

Comment:
Can you add a comment here indicating the expectations behind when/where/how the CI variable is defined?

Reply:
I guess it would also be good to add a note (or issue reference) to explain why generating these .bc files were moved into the CI-protected section.

Reply:
I've added a comment which should address both of your requests. (Let me know if that is not the case.)